DOCS.md

```diff
@@ -45,4 +45,35 @@ deploy:
 
 It's not recommended to update multiples of namespaces, deployment_names, and conatiner_names.  Try to keep things simple.
 
-if you want to add secrets for the token it's KUBERNETES_TOKEN
+If you want to add secrets for the token or url use KUBERNETES_TOKEN, KUBERNETES_URL
+
+```yaml
+pipeline:
+  deploy:
+    image: peloton/drone-k8s-deployment
+    token: asldkfj
+    insecure: false
+    deployment_names: [mything, mything2]
+    container_names: mything
+    namespaces: [mynamespace, anothernamspace]
+    docker_image: drone/drone:latest
+    secrets: [kubernetes_url, kubernetes_token]
+```
+
+If you want to force docker container upgrading, there is a hack which allows you to do this even if your image name/tag has not changed. Put to the data_label key ENV variable from Drone CI with a timestamp, such as this:
+
+```yaml
+pipeline:
+  deploy:
+    image: peloton/drone-k8s-deployment
+    token: asldkfj
+    insecure: false
+    deployment_names: [mything, mything2]
+    container_names: mything
+    namespaces: [mynamespace, anothernamspace]
+    docker_image: drone/drone:latest
+    date_label: "${DRONE_BUILD_FINISHED}"
```

Comment:
This should be like date_label: deployment.drone.io/date-deployed :) The date gets added to whatever annotation you use :)

Reply:
silly question, but how does it work? :) Do you mean that "deployment.drone.io/date-deployed" will automatically add the date of the deploy?
I have not seen this in drone ci documentation

Reply:
no, the plugin adds the current date/time to whatever label you give it (not saying it's the best implementation but it's how it currently works) might be better in the future to have a map of the key and what the value should be like DRONE_BUILD_CREATED. I just didn't do it that way yet :)